app/script/strings.js

```diff
@@ -179,7 +197,141 @@ export const chooseHandleStrings = defineMessages({
+  },
+  subhead: {
+    id: 'conversationJoin.subhead',
+    defaultMessage: 'Encrypted group messaging and conference calls. <br />No account necessary.',
+  },
+  existentAccountHeadline: {
+    id: 'conversationJoin.existentAccountHeadline',
+    defaultMessage: '{name}, you have been invited <br />to join a <b style="color: black;">Wire guest room.</b>',
+  },
+  existentAccountSubhead: {
+    id: 'conversationJoin.subheadExistentAccount',
+    defaultMessage: 'Guest rooms let you have conversations with people who are not on your team.',
+  },
+  existentAccountOpenButton: {
+    id: 'conversationJoin.buttonExistentAccountOpen',
+    defaultMessage: 'Open in Wire',
+  },
+  existentAccountJoinWithoutLink: {
+    id: 'conversationJoin.existentAccountJoinWithoutLink',
+    defaultMessage: 'Join the conversation',
+  },
+  existentAccountJoinWithoutText: {
+    id: 'conversationJoin.existentAccountJoinWithoutText',
+    defaultMessage: 'without an account.',
+  },
+  invalidHeadline: {
+    id: 'conversationJoin.invalidHeadline',
+    defaultMessage: 'This <b style="color: black;">Wire guest room</b><br />is now closed.',
+  },
+  invalidSubhead: {
+    id: 'conversationJoin.invalidSubhead',
+    defaultMessage: 'Ask the person who invited you how to join.',
+  },
+  invalidCreateAccountLink: {
+    id: 'conversationJoin.invalidCreateAccountLink',
+    defaultMessage: 'Create an account',
+  },
+  invalidCreateAccountText: {
+    id: 'conversationJoin.invalidCreateAccountText',
+    defaultMessage: 'for group messaging and conference calls.',
+  },
+  fullConversationHeadline: {
+    id: 'conversationJoin.fullConversationHeadline',
+    defaultMessage: 'This <b style="color: black;">Wire guest room</b><br />is full.',
+  },
+  fullConversationSubhead: {
+    id: 'conversationJoin.fullConversationSubhead',
+    defaultMessage: 'Talk to the person who invited you.',
+  },
+  namePlaceholder: {
+    id: 'conversationJoin.namePlaceholder',
+    defaultMessage: 'Your name',
+  },
+  hasAccount: {
+    id: 'conversationJoin.hasAccount',
+    defaultMessage: 'Already have an account?',
+  },
+  loginLink: {
+    id: 'conversationJoin.loginLink',
+    defaultMessage: 'Log in',
+  },
+  acceptTou: {
+    id: 'conversationJoin.acceptTou',
+    defaultMessage: 'By continuing you accept the Wire',
+  },
+  touLink: {
+    id: 'conversationJoin.touLink',
+    defaultMessage: 'Terms of Use',
+  },
+});
+
 export const errorHandlerStrings = defineMessages({
+  [BackendError.LABEL.CONVERSATION_CODE_NOT_FOUND]: {
+    id: 'BackendError.LABEL.CONVERSATION_CODE_NOT_FOUND',
+    defaultMessage: 'CONVERSATION_CODE_NOT_FOUND',
+  },
+  [BackendError.LABEL.CONVERSATION_NOT_FOUND]: {
+    id: 'BackendError.LABEL.CONVERSATION_NOT_FOUND',
+    defaultMessage: 'CONVERSATION_NOT_FOUND',
+  },
+  [BackendError.LABEL.CONVERSATION_TOO_MANY_MEMBERS]: {
+    id: 'BackendError.LABEL.CONVERSATION_TOO_MANY_MEMBERS',
+    defaultMessage: 'CONVERSATION_TOO_MANY_MEMBERS',
```

Comment:
Add error messages

Reply:
These messages are not visible for users currently since we have separate pages for these cases and only check for the error type.